src/themes/capybara/components/category/SubCategoriesSidebar.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <SfAccordion :first-open="true">
+    <template v-slot="{ selected }">
+      <SfAccordionItem
+        v-for="(parentCategory, i) in filterCategories(currentCategory.level+1, currentCategory.id)"
+        :key="i"
+        :header="parentCategory.name"
+      >
+        <template #content>
+          <SfList>
+            <SfListItem v-for="(category, j) in filterCategoryChildren(parentCategory.id)" :key="j">
+              <SfMenuItem
+                class="menu-item"
+                :label="category.name"
+              />
+            </SfListItem>
+          </SfList>
+        </template>
+      </SfAccordionItem>
+    </template>
+  </SfAccordion>
+</template>
+
+<script>
+import { SfAccordion, SfList, SfMenuItem } from '@storefrontui/vue';
+
+export default {
+  props: {
+    categories: {
+      type: Array,
+      default: () => []
+    },
+    currentCategory: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  methods: {
+    filterCategories (level, parentId) {
+      return this.categories.filter(category => category.level === level && category.parent_id === parentId)
+    },
+    filterCategoryChildren (parentId) {
```

Comment:
Will this work with dynamic category pre-fetching? We don't have full categories tree loaded at once. The sub categories should be loaded once user clicked the parent category to save the `__INITIAL_STATE__` size: it has been introduced with https://github.com/DivanteLtd/vue-storefront/pull/2100

On sites like `wonect.com` the category tree itslelf is 2MB we can't render it into HTML @patzick was dealing with this with the category refactor as well

Reply:
We should load just the root categories at first - display the first level of menu

Reply:
@pkarw here (according to designs) categories are only 2 levels deep (based on which category we are because those are subcategories of the current one)